prov/verbs/src/verbs_domain.c

```diff
@@ -309,6 +319,18 @@ static struct fi_ops_domain vrb_dgram_domain_ops = {
 };
 
 
+static void vrb_domain_enable_async_events(struct vrb_domain *domain)
+{
+	assert(domain->verbs);
+	int ret = fi_fd_nonblock(domain->verbs->async_fd);
```

Comment:
I would change this to be
```
int ret;

assert(domain->verbs);
ret = fi_fd_nonblock(domain->verbs->async_fd);
```
which matches the declaration at the top, new line, then code format